libkpod/config.go

```diff
@@ -23,7 +23,7 @@ const (
 	cniBinDir           = "/opt/cni/bin/"
 	cgroupManager       = oci.CgroupfsCgroupsManager
 	lockPath            = "/run/crio.lock"
-	containerExitsDir   = "/var/run/kpod/exits"
+	containerExitsDir   = "/var/run/crio/exits"
```

Comment:
We should get this out of cri configs?  Not hard coded

Reply:
Perhaps this should be a generic directory like /var/run/containers/exits

Reply:
@mrunalp added it specifically to kpod ... if there is no need now or later to define it to be different than crio, I would agree.

Reply:
The problem with this now, it is would not be able to be tests, since rests override the /var/run/crio directory.

Reply:
Ok, relocated the exitdir declaration to oci.go